openpype/tools/sceneinventory/switch_dialog.py

```diff
@@ -1227,12 +1227,12 @@ def _trigger_switch(self, loader=None):
 
         version_ids = list()
 
-        version_docs_by_parent_id = {}
```

Comment:
I would probably keep it as `version_docs_by_parent_id` and add a tuple in the value field:
```py
version_docs_by_parent_id = collections.defaultdict(dict)

for version_doc in versions_docs:
	version_ids.append(version_doc["id"])
	version_docs_by_parent_id[version_doc["parent"]] = (
		version_doc["name"],
		version_doc
	)
```


or even:
```py

versions_ids = [
	version_doc["id"]
	for version_doc in version_docs
]

version_docs_by_parent_id = {
	version_doc["parent"]: (version_doc["name"], version_doc)
	for version_doc in version_docs
}
```

Then down below:
```py
version_name, version_doc = version_docs_by_parent_id[subset_id]
...
```

Reply:
That won't help, you can have multiple versions under single subset.